### PR TITLE
Implement calendar

### LINKED ISSUE #134: As a store manager, I want to have a calendar in the application
... so that I can easily know which day is it today and plan for work schedule more conveniently.

### PR DESCRIPTION
**Pull request by yanbingtao**
Now, a working calendar GUI can be displayed in the calendar pane.

### CODE REVIEW

File: src/main/java/seedu/address/ui/CalendarView.java [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+/**
+ * Calendar View UI.
+ */
+public class CalendarView extends UiPart<Region> {
+
+    private static final String FXML = "CalendarView.fxml";
+    private static final String[] MONTHS = {"January", "February", "March", "April", "May", "June",
+        "July", "August", "September", "October", "November", "December"};
+    private static final int TOTALNUMOFDATEGRID = 42;
+    private static final int NUMOFDAYSPERWEEK = 7;
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/ui/CalendarView.java]: Should add underscore between the words. I believe this is part of the naming conventions

**Review by ureshiiYing — COMMENTED** [review]
The actual GUI looks good to me. There's some problems in the coding standards that I have commented.

File: src/main/java/seedu/address/ui/CalendarView.java (hunk 1 above)
Comment (yanbingtao) [reply src/main/java/seedu/address/ui/CalendarView.java]: Good point! I've changed these 2 in the latest commit. Thank you!

**Comment by yanbingtao** [comment]
> The actual GUI looks good to me. There's some problems in the coding standards that I have commented.

Thank you! I've edited in the latest commit. Thank you!

**Review by Diwu-Yi — APPROVED** [review]
LGTM

**Review by Persdre — APPROVED** [review]
LGTM

**Pull request merged**